src/frontend/components/interfaces/AddInterfacePage.vue

```diff
@@ -217,6 +217,8 @@ export default {
                     125000, // 125 kHz
                     250000, // 250 kHz
                     500000, // 500 kHz
+                    1625000, // 1625000 kHz for 2.4 GHz Sx1280
```

Comment:
its a typo. 1625 kHz is the correct number in the comment section

Reply:
No worries, will fix after merge.